js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -24,92 +99,231 @@ export const AssetAutomaterializePolicyPage = () => {
+      const max = data.assetNodeOrError.autoMaterializePolicy?.maxMaterializationsPerMinute;
+      if (typeof max === 'number') {
+        setMaxMaterializationsPerMinute(max);
+      }
+    }
+  }, [data, setMaxMaterializationsPerMinute]);
 
-const RightPanel = () => {
   return (
-    <Box flex={{direction: 'column'}} style={{maxWidth: '294px'}}>
+    <Box flex={{direction: 'column'}} style={{width: '294px'}}>
       <Box
         padding={{vertical: 16, horizontal: 24}}
         border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
       >
         <Subheading>Overview</Subheading>
       </Box>
-      <RightPanelSection title="Auto-materialize Policy">
-        <RightPanelDetail
-          title="On missing"
-          value={<Green700Text>True</Green700Text>}
-          tooltip="test"
-        />
-        <RightPanelDetail
-          title="On upstream data"
-          value={<Green700Text>True</Green700Text>}
-          tooltip="test"
-        />
-        <RightPanelDetail
-          title="For freshness"
-          value={<Green700Text>True</Green700Text>}
-          tooltip="test"
-        />
-        <RightPanelDetail title="Maximum materializations per hour" value={2} tooltip="test" />
-      </RightPanelSection>
-      <RightPanelSection title="Freshness Policy">
-        <RightPanelDetail title="Maximum lag minutes" tooltip="test" value={2} />
-        <Box flex={{direction: 'column', gap: 8}}>
-          This asset will be considered late if it is not materialized within 2 minutes of it’s
-          upstream dependencies.
-          <ButtonLink>View upstream assets</ButtonLink>
+      {error ? (
+        <Box padding={24}>
+          <ErrorWrapper>{JSON.stringify(error)}</ErrorWrapper>
         </Box>
-      </RightPanelSection>
+      ) : !data ? (
+        <Box flex={{direction: 'row', justifyContent: 'center'}} padding={{vertical: 24}}>
+          <Spinner purpose="section" />
+        </Box>
+      ) : data.assetNodeOrError.__typename === 'AssetNotFoundError' ? (
+        <Redirect to="/assets" />
+      ) : (
+        <>
+          {data.assetNodeOrError.autoMaterializePolicy ? (
+            <RightPanelSection
+              title={
+                <Box
+                  flex={{direction: 'row', justifyContent: 'space-between', alignItems: 'center'}}
+                >
+                  Auto-materialize Policy
+                  <AutomaterializePolicyTag policy={data.assetNodeOrError.autoMaterializePolicy} />
+                </Box>
+              }
+            >
+              <Body style={{flex: 1}}>
+                {automaterializePolicyDescription(data.assetNodeOrError.autoMaterializePolicy)}
+              </Body>
+            </RightPanelSection>
+          ) : (
+            <NonIdealState
+              title="No Automaterialize policy found"
+              shrinkable
+              description={
+                <Box flex={{direction: 'column', gap: 8}}>
```

Comment:
Is this also found on the asset Definition page still? If so maybe we should make these empty state components re-usable so we don't change the text in one place and forget it in the other.

Reply:
I'm not sure?